src/app/move-popup/dimMoveItemProperties.html

```diff
@@ -49,7 +49,8 @@
       <form ng-if="vm.item.taggable" name="notes"><textarea name="data" ng-i18next="[placeholder]Notes.Help" ng-maxlength="120" ng-model="vm.item.dimInfo.notes" ng-model-options="{ debounce: 250 }" ng-change="vm.updateNote()"></textarea></form>
       <span class="textarea-error" ng-show="notes.data.$error.maxlength" ng-i18next="Notes.Error"></span>
       <div class="item-description" ng-if="vm.showDescription" ng-bind="::vm.item.description"></div>
-      <div class="masterwork-progress" ng-if="vm.item.masterwork" ng-i18next="[html:i18next]({ progress: '<strong>' + vm.masterworkProgress() + '</strong>' })ItemService.MasterworkProgress"></div>
+      <div class="masterwork-progress" ng-if="vm.item.masterwork && vm.masterworkType() == 'Vanguard'" ng-i18next="[html:i18next]({ progress: '<strong>' + vm.masterworkProgress() + '</strong>' })ItemService.MasterworkProgressVanguard"></div>
+      <div class="masterwork-progress" ng-if="vm.item.masterwork && vm.masterworkType() == 'Crucible'" ng-i18next="[html:i18next]({ progress: '<strong>' + vm.masterworkProgress() + '</strong>' })ItemService.MasterworkProgressCrucible"></div>
```

Comment:
We shouldn't call methods inside Angular bindings - they'll get re-run on every digest, which can be incredibly expensive! If you want to compute a value off of this, $watch the property you're interested in and recompute it onto a simple property on the view model.